Firestore/Source/API/FSTUserDataConverter.mm

```diff
@@ -424,9 +424,46 @@ - (FSTParsedSetData *)parsedMergeData:(id)input {
                                     path:absl::make_unique<FieldPath>(FieldPath::EmptyPath())];
   FSTObjectValue *updateData = (FSTObjectValue *)[self parseData:input context:context];
 
+  FieldMask convertedFieldMask;
+  std::vector<FieldTransform> convertedFieldTransform;
+
+  if (fieldMask) {
+    __block std::vector<FieldPath> fieldMaskPaths{};
+    [fieldMask
+        enumerateObjectsUsingBlock:^(id _Nonnull fieldPath, NSUInteger idx, BOOL *_Nonnull stop) {
```

Comment:
I don't believe you need the `_Nonnull` annotations, as you are within an `NS_ASSUME_NONNULL` block.

Reply:
Done.